### ISSUE TITLE
Boston Light Swim issues

### ISSUE THREAD

**Issue by rsignell-usgs** [ioos/notebooks_demos#109]
* I ran step 0 (fetch data) on Windows/python 3 one time and got COAWST, and ran it another time and didn't get COAWST.  Are we getting bit by the fact that pyCSW only returns the first 10 records if we don't iterate?  

* The FVCOM NECOFS models are being discovered, but the data not picked up.  We are getting this message in the logs: 
```
 [Reading url 1/3]: http://www.smast.umassd.edu:8080/thredds/dodsC/FVCOM/NECOFS/Forecasts/NECOFS_GOM3_FORECAST.nc
07:51:24 WARNING: Cannot get cube for: http://www.smast.umassd.edu:8080/thredds/dodsC/FVCOM/NECOFS/Forecasts/NECOFS_GOM3_FORECAST.nc
list index out of range
07:51:24 INFO: 
[Reading url 2/3]: http://oos.soest.hawaii.edu/thredds/dodsC/pacioos/hycom/global
07:51:27 INFO: 
[Reading url 3/3]: http://www.smast.umassd.edu:8080/thredds/dodsC/FVCOM/NECOFS/Forecasts/NECOFS_FVCOM_OCEAN_MASSBAY_FORECAST.nc
07:51:47 WARNING: Cannot get cube for: http://www.smast.umassd.edu:8080/thredds/dodsC/FVCOM/NECOFS/Forecasts/NECOFS_FVCOM_OCEAN_MASSBAY_FORECAST.nc
list index out of range
```

* Step 1 (skill score) is dying with this message: 
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-3c9d8de37e0b> in <module>()
     13 # Pretty HTML table.
     14 df = rename_cols(df, config)
---> 15 html = to_html(df.T)
     16 fname = os.path.join(save_dir, 'mean_bias.html')
     17 save_html(fname, html)

C:\Users\rsignell\Documents\GitHub\notebooks_demos\ioos_tools\ioos.py in to_html(df, css)
    625     with open(css, 'r') as f:
    626         style = """<style>{}</style>""".format(f.read())
--> 627     table = dict(style=style, table=df.to_html())
    628     return HTML('{style}<div class="datagrid">{table}</div>'.format(**table))
    629 

C:\Users\rsignell\AppData\Local\Continuum\Miniconda3\envs\ioos3\lib\site-packages\pandas\core\generic.py in __getattr__(self, name)
   2742             if name in self._info_axis:
   2743                 return self[name]
-> 2744             return object.__getattribute__(self, name)
   2745 
   2746     def __setattr__(self, name, value):

AttributeError: 'Series' object has no attribute 'to_html'
```
It looks like this is because we only got one model output, so got a Series when we expected a  Dataframe.   So maybe when the NECOFS data comes back, this error will go away.  But we probably should eventually handle the case where there is only one model result. 

* There was something that triggered a problem with Step 0 where it died when it tried to remove the latest directory, something about a permissions problem.    But then I just ran the cell again and it worked.   I was unable to recreate this, but just putting this in here in case it happens again or triggers an idea.

**Comment by ocefpaf** [comment]
> I ran step 0 (fetch data) on Windows/python 3 one time and got COAWST, and ran it another time and didn't get COAWST. Are we getting bit by the fact that pyCSW only returns the first 10 records if we don't iterate?

It is possible but I haven't addressed that in #110 yet because I found other issues and I want o isolate the PRs to be sure to fix them.

> The FVCOM NECOFS models are being discovered, but the data not picked up. We are getting this message in the logs:

I tried to be smart in https://github.com/ioos/notebooks_demos/pull/103 to avoid a huge `coord` data download but it seems that I introduced a bug. Should be fixed in #110.

> Step 1 (skill score) is dying with this message: AttributeError: 'Series' object has no attribute 'to_html'
It looks like this is because we only got one model output, so got a Series when we expected a Dataframe. So maybe when the NECOFS data comes back, this error will go away. But we probably should eventually handle the case where there is only one model result.

I added an `if-clause` to catch that. It meas we do not have any skill values to show.

**Comment by rsignell-usgs** [comment]
Woohoo!  Way to go @ocefpaf:
![2016-11-09_9-43-04](https://cloud.githubusercontent.com/assets/1872600/20142139/8c76ae90-a661-11e6-815e-caec3a4b66d1.png)
